cogs/tags/tags.py

```diff
@@ -266,6 +269,15 @@ def checkAliasCog(self, name: str = None):
                 )
         return aliasCog
 
+    def checkValidCommandName(self, name: str):
+        """Check to see if we can use the name as a tag"""
+        if name in self.tagCommands:
+            raise ReservedTagNameError("This tag name is reserved, please use a different name!")
+        if any(char.isspace() for char in name):
```

Comment:
Couldn't this just be 
`if(` " " in name)`

Reply:
Ah yeah that would work, too. I opted to use the method in customcom since we don't have to explicitly say `" "`.